<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraClass</class>
 <widget class="QWidget" name="cameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageCapture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="camera.qrc">
    <normaloff>:/Resources/UltimageTK_LogoOnly.png</normaloff>:/Resources/UltimageTK_LogoOnly.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="listViewVideo"/>
     </item>
     <item>
      <widget class="QListView" name="listViewImage"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnOpenCamera">
         <property name="text">
          <string>Open Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCloseCamera">
         <property name="text">
          <string>Close Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStartRec">
         <property name="text">
          <string>Start Rec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEndRec">
         <property name="text">
          <string>End Rec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnScreenShots">
         <property name="text">
          <string>Screen Shots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLanguage">
         <item>
          <property name="text">
           <string>Chinese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="camera.qrc"/>
 </resources>
 <connections/>
</ui>
